trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -132,16 +132,22 @@ base16(<TraceOptions>) = 00  // not-sampled
 
 # TraceState field
 
+The `tracestate` HTTP header field conveys information about request position in multiple distributed tracing graphs.
+
 ## Header name
 
 `tracestate`
 
 ## Header value
 
-`vendorName1=opaqueValue1,vendorName2=opaqueValue2`
+`tracestate` is a Structured header in accordance to [Draft of Structured Headers for HTTP](http://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html). Its value MUST be a dictionary ([Draft of Structured Headers for HTTP](http://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html), Section 4.1).
+
+Maximum length of a combined header MUST be less than 512 bytes. If the maximum length of a combined header is more than 512 bytes it SHOULD be ignored.
 
-The value a concatenation of trace graph name-state pairs. Only one entry per
-name is allowed because the entry represents that last position in the trace.
+Example: `vendorName1=opaqueValue1,vendorName2=opaqueValue2`
+
+The value a concatenation of trace graph key-value pairs. Only one entry per
+key is allowed because the entry represents that last position in the trace.
```

Comment:
This is already defined by using a dictionary

Reply:
you commenting to outdated text

Reply:
I think you missed discussion in #81